vector/v.to.rast/dense_line.c

```diff
@@ -274,3 +274,15 @@ void dense_line(double x1, double y1, double x2, double y2,
     }
     point(ix2, iy2);
 }
+
+/* point plotting, alternative to G_plot_point()
+ * x, y are col, row numbers */
+void plot_point(double east, double north)
```

Comment:
If it is an alternative, does that mean that if changes to the G_plot_point() are made, they may need to be kept in sync/compatible with the function here? If so, should we add a note in the comments of G_plot_point() to refer back here (to not forget in the future)?

Reply:
That's a good idea.